src/core/iomgr/pollset_posix.c

```diff
@@ -161,8 +173,8 @@ void grpc_pollset_del_fd(grpc_pollset *pollset, grpc_fd *fd) {
 
 int grpc_pollset_work(grpc_pollset *pollset, gpr_timespec deadline) {
   /* pollset->mu already held */
-  gpr_timespec now;
-  now = gpr_now();
+  gpr_timespec now = gpr_now();
```

Comment:
So I've tended to do this form of initialization wherever it's legal - mostly as a code density improvement to ease scanning.